packages/errors/src/errors.ts

```diff
@@ -1309,10 +1309,28 @@ export const AllCypressErrors = {
     `
   },
 
-} as const
+  MIGRATED_OPTION_INVALID: (optionKey: string, err?: Error) => {
```

Comment:
nit: can we make this name a little more specific. Maybe `SETUP_NODE_EVENTS_RESOLVED_CONFIG_INVALID`

Reply:
You got it